pkg/utils/docker/image.go

```diff
@@ -68,10 +68,20 @@ func GetWorkerImage(client client.Client, datasetName string, runtimeType string
 
 	}
 	if imageName == "" {
-		imageName = "registry.cn-huhehaote.aliyuncs.com/alluxio/alluxio"
+		if runtimeType == common.ALLUXIO_RUNTIME {
```

Comment:
I suggest to use switch here.

Reply:
I and @XIAO-HOU are trying to [unify get image methods](https://github.com/fluid-cloudnative/fluid/pull/728). 
perhaps you need to agree with us in the future.